docs/en/writing-running-appium/android/android-screen-streaming.md

```diff
@@ -50,6 +50,16 @@ driver.execute_script('mobile: startScreenStreaming', {
 })
 ```
 
+```javascript
+// Javascript
```

Comment:
can you say which javascript library you are using? is it webdriverio? would be good to put that in this comment as well.